_posts/en/newsletters/2020-09-09-newsletter.md

```diff
@@ -0,0 +1,159 @@
+      the highest-fee replacement after they've finished receiving their
+      new UTXO.  This won't earn them any money, but the high fees will
+      be deducted from the value of your UTXO and so there's no cost to
+      the attacker.
+
+    - *Costless pinning theft:* like above, if there are multiple
+      versions of the transaction with different feerates, an attacking
+      counterparty can broadcast a low-feerate version and then use a
+      [transaction pinning][topic transaction pinning] method to make it
+      impractical to fee bump.  If the attack is successful, it would
+      allow the attacker to spend the victim's money; if it was
+      unsuccessful, it might cost the attacker nothing extra if they
+      were already planning to send the transactions they used for
+      pinning (e.g. they were planning to consolidate UTXOs anyway).
+
+  Several potential solutions to these attacks were discussed.  The
+  [most promising solution][belcher collateral] was requiring anyone
+  spending a coinswap UTXO to spend part of the value of one of their
+  other UTXOs to transaction fees.  This *collateral payment* would
+  eliminate the costless aspect of the previously described attacks.
+  Fully addressing the attacks might also require some other changes
+  related to timelocks, such as using a 1-block relative timelock ("`1
+  OP_CSV`") to prevent pinning.
+
+  In addition to the discussion helping to improve a privacy-enhancing
+  coinswap protocol, at least one participant [noted][zmnscpxj ln] that
+  some of the insights from the conversation might be applicable to
+  other protocols, such as LN.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+{% include functions/details-list.md
+  q0="FIXME"
+  a0="FIXME"
+  a0link="https://bitcoincore.reviews/FIXME"
+%}
+
+FIXME:jnewbery or jonatack
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update on Tuesday
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19405][] rpc, cli: add network in/out connections to `getnetworkinfo` and `-getinfo` FIXME: bitschmidty
+
+- [Bitcoin Core #19670][] Protect localhost and block-relay-only peers from eviction FIXME:dongcarl
+
+- [Bitcoin Core #14687][] updates the zmq event notification service to
```

Comment:
Seems zmq is normally written as ZeroMQ (also spelled ØMQ, 0MQ or ZMQ), source https://en.wikipedia.org/wiki/ZeroMQ.

In bitcoin/bitcoin, `doc/zmq.md` calls it ZeroMQ.